src/SecurityChecker/Command/SecurityCheckerCommand.php

```diff
@@ -1,12 +1,157 @@
+    {
+        $this->loadFile();
+        $this->changeFileMode();
+        $commandOutput = $this->runCommand();
+        $commandOutput = $this->markFalsePositiveResults($commandOutput);
+
+        $result = $this->createResultFromCommandOutput($commandOutput);
+
+        $output->writeln($result->getVulnerabilities());
+
+        return $this->convertResultToExitCode($result, $output);
+    }
+
+    /**
+     * @return void
+     */
+    protected function loadFile(): void
+    {
+        $process = new Process(['wget', static::BINARY_CHECKER, '-O', static::FILE_NAME]);
+        $process->run();
+    }
+
+    /**
+     * @return void
+     */
+    protected function changeFileMode(): void
+    {
+        chmod(static::FILE_NAME, 0777);
+    }
+
+    /**
+     * @throws \Symfony\Component\Process\Exception\ProcessFailedException
+     *
+     * @return string
+     */
+    protected function runCommand(): string
+    {
+        $process = new Process([static::FILE_NAME, 'check:security']);
+        $process->run();
+
+        if (!empty($process->getErrorOutput())) {
+            throw new ProcessFailedException($process);
+        }
+
+        return $process->getOutput();
+    }
+
+    /**
+     * @param string $commandOutput
+     *
+     * @return \SecurityChecker\Result
+     */
+    protected function createResultFromCommandOutput(string $commandOutput): Result
+    {
+        $count = 0;
+        preg_match('/\d\spackage/m', $commandOutput, $matches);
+        if (!empty($matches)) {
+            $count = (int)$matches[0];
+        }
+
+        return new Result($count, $commandOutput);
+    }
+
+    /**
+     * @param string $commandOutput
+     *
+     * @return string
+     */
+    protected function markFalsePositiveResults(string $commandOutput): string
+    {
+        if (strpos($commandOutput, static::FALSE_POSITIVE_ISSUE_NUMBER) !== false) {
+            $commandOutput = str_replace(
+                static::FALSE_POSITIVE_ISSUE_NUMBER,
+                sprintf('<info>%s - is a false positive</info>', static::FALSE_POSITIVE_ISSUE_NUMBER),
+                $commandOutput
+            );
+        }
+
+        return $commandOutput;
+    }
+
+    /**
+     * @param \SecurityChecker\Result $result
+     * @param \Symfony\Component\Console\Output\OutputInterface $output
+     *
+     * @return int
+     */
+    protected function convertResultToExitCode(Result $result, OutputInterface $output): int
+    {
+        if ($result->count() === 0) {
+            return 0;
+        }
+
+        if ($this->checkResultForFalsePositiveCase($result)) {
+            if ($output->isVerbose()) {
+                $output->writeln(sprintf('<info>The issue about %s is a false positive result</info>', static::FALSE_POSITIVE_ISSUE_NUMBER));
+                $output->writeln('<info>Check https://github.com/FriendsOfPHP/security-advisories/issues/511 for details</info>');
+            }
+
+            return 0;
+        }
+
+        return 1;
```

Comment:
Let's apply "Early return" pattern here.
```suggestion
       if (!$this->checkResultForFalsePositiveCase($result)) {
            return 1;
        }

        if ($output->isVerbose()) {
            $output->writeln(sprintf('<info>The issue about %s is a false positive result</info>', static::FALSE_POSITIVE_ISSUE_NUMBER));
            $output->writeln('<info>Check https://github.com/FriendsOfPHP/security-advisories/issues/511 for details</info>');
        }

        return 0;
```